### PR TITLE
Adding more detail to the README

### PR DESCRIPTION
**Pull request by sgrichards**
Updating readme to clarify the the concepts of workspace/harness and include some universal commands.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,26 +5,66 @@ Workspace is a tool to orchestrate and bring consistency to your project environ
...
+ * Wordpress
+ 
+## Commands
+
+By default a workspace project ships with the following commands:
+
+| Command                                    | Description                                             |
+|---                                         |---                                                      |
+|`ws install`                                |Install project                                          |
+|`ws enable`                                 |Start a previously disabled environment                  |
```
Comment (kierenevans) [inline README.md:66]: This project only defines:
```
ws

Sub Commands:
   global       
   secret       
   install      
   harness      

Global Options:
  -h, --help    Show help message
```
```
ws global service <name> <action> [options]
ws global config get <key>
ws secret encrypt <message> [<key>] [options]
ws secret decrypt <encrypted> [options]
ws secret generate-random-key

ws install

Usage:
  ws install [options]


Command Options:
      --step=<step>    Step from which to start installer.
      --skip-events    If set events will not be triggered.

ws harness download
ws harness prepare
```
The rest are from harnesses, and may not exist.

File: README.md [hunk 2]
```diff
@@ -5,26 +5,66 @@ Workspace is a tool to orchestrate and bring consistency to your project environ
...
-```
-### Anatomy of a workspace
 
 #### Key Concepts
- - [Workspace](docs/concepts/workspace.md)
- - [Harness](docs/concepts/harness.md)
+
+##### Workspace
+ 
+`workspace` describes both the CLI tool `ws` and the project environment itself. It is dependent on a harness definition.
```
Comment (jameshalsall) [inline README.md]: it is not dependent on a harness definition

File: README.md [hunk 3]
```diff
@@ -5,26 +5,66 @@ Workspace is a tool to orchestrate and bring consistency to your project environ
...
+ 
+##### Harness
  
+ A harness is a "flavour" of workspace which defines the blueprint to which a project environment will adhere. It  
+ incorporates two distinct elements:
+ 
+ 1. *Skeleton* - a project stub applied when the workspace is first created with a given harness
```
Comment (kierenevans) [inline README.md:37]: These are both "harness" concepts unrelated to the workspace tool, and may not be supported by all harnesses - perhaps we could document this in the harness repos and link to it from here?
e.g.
* https://github.com/inviqa/harness-base-php
* https://github.com/inviqa/harness-keycloak
* https://github.com/inviqa/harness-node
and maybe a link to https://my127.io/workspace/harnesses.json ?

Comment (coledave) [reply README.md:37]: :+1: most docs are going to be harness specific, we can flesh out the basic types and concepts here and better explain its more like a task runner with templating where you have shared harnesses to give you a quick start when creating a project.

Comment (sgrichards) [reply README.md:37]: Agreed - there's definitely value in summarising the Harness concept here and signposting to the detail

File: README.md (hunk 1 above)
Comment (sgrichards) [reply README.md:66]: Is it fair to say most harnesses implement these universal commands?

Comment (sgrichards) [reply README.md:66]: At this point i'd suggest using `ws` without a harness is the edge case

**Pull request closed without merging**